docs/protocol/concepts/gas-and-fees.md

```diff
@@ -4,7 +4,19 @@ sidebar_position: 4
 
 # Gas and Fees
 
-Learn about the differences between `Gas` and `Fees` in Ethereum and Cosmos.
+Users need to pay a fee to submit transactions on the Evmos network.
+As fees are handled differently on Ethereum and Cosmos,
+it is important to understand how the Evmos blockchain implements an Ethereum-type fee calculation,
+that is compatible with the Cosmos SDK.
+
+Therefore this overview explains the basics of gas calculation,
+how to provide fees for transactions
+and how the Ethereum-type fee calculation uses a fee market (EIP1559)
+for prioritizing transactions.
+
+Also, note the fees that are paid for interacting with smart contracts on Evmos
+can earn smart contract deployers a revenue. For information on this,
+head to the [develop](../../develop/mainnet/index.md#revenue).
```

Comment:
the link does not work, you will need to remove the `index.md` part and the forward slash after `mainnet` and that should work